examples/evaluation/evaluate_on_regressor.py

```diff
@@ -0,0 +1,30 @@
+import mlflow
+from sklearn.datasets import load_boston
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+
+mlflow.sklearn.autolog()
+
+boston_data = load_boston()
```

Comment:
Can we avoid using `load_boston` since it's deprecated?

https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html

Reply:
Created a PR for this.